addons/web_editor/static/src/js/widgets.js

```diff
@@ -1143,7 +1154,7 @@ var LinkDialog = Dialog.extend({
     },
     onkeyup: function (e) {
         var $e = $(e.target);
-        var is_link = ($e.val()||'').length && $e.val().indexOf("@") === -1;
+        var is_link = ($e.val()||'').length >= 0 && $e.val().indexOf("@") === -1;
```

Comment:
If `input.window-new` must always be visible, then remove all code which hides/shows it :)